indexing-service/src/main/java/io/druid/indexing/common/task/HadoopIndexTask.java

```diff
@@ -68,6 +71,9 @@ private static String getTheDataSource(HadoopIngestionSpec spec)
   @JsonIgnore
   private final ObjectMapper jsonMapper;
 
+  @JsonIgnore
+  private Interval interval;
```

Comment:
I think it could be a local variable in 'run', it looks like it's just needed to be remembered so we can upgrade later.